### PR TITLE
feat(examples): add `r/morgan/{home,guestbook}`

### PR DESCRIPTION
**Pull request by thehowl**
I wanted a simple realm that allows any user to interact with it and see something happen. This is a guestbook realm; it allows anyone to leave a small message, which will be associated with their address.

It also is a nice showcase of how to effectively do pagination with AVL trees. I don't think we have many such examples so far.

### CODE REVIEW

**Comment by thehowl** [comment]
Leon pointed out that we have [r/demo/userbook](https://github.com/gnolang/gno/commits/master/examples/gno.land/r/demo/userbook), which achieves something similar. We can consider whether to keep both, or just one or the other.

I see two advantages with this realm: using avl.Trees + pagination, and providing users with a way to actually "write something" (with the consequence of moderation -- hence why there is an admin function the remove a user's post; though obviously with the faucet working anyone can just have another account).

cc @moul

File: examples/gno.land/r/demo/guestbook/admin.gno [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"gno.land/p/demo/seqid"
+)
+
+var adminAddr = std.PrevRealm().Addr()
+
+// AdminDelete removes the guestbook message with the given ID.
+// The user will still be marked as having submitted a message, so they
+// won't be able to re-submit a new message.
+func AdminDelete(signatureID string) {
+	if std.PrevRealm().Addr() != adminAddr {
```
Comment (leohhhn) [inline examples/gno.land/r/demo/guestbook/admin.gno:15]: Let's use `p/demo/ownable`?

**Review by moul — APPROVED** [review]
Looks good, but we should consider renaming this example to avoid conflicts with other guestbook implementations.

**Comment by thehowl** [comment]
Moved it to r/morgan/guestbook and added a small home realm. It seems appropriate for now.

File: examples/gno.land/r/morgan/guestbook/guestbook_test.gno [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	}
+	if guestbook.Size() != 2 {
+		t.Error("invalid guestbook size")
+	}
+}
+
+func TestSign_FromRealm(t *testing.T) {
+	std.TestSetRealm(std.NewCodeRealm("gno.land/r/demo/users"))
+
+	defer func() {
```
Comment (leohhhn) [inline examples/gno.land/r/morgan/guestbook/guestbook_test.gno:38]: btw, we have `urequire` & `uassert` if you want to try to use them. Up to you though

**Review by leohhhn — APPROVED** [review]

**Pull request merged**